paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+
+![bboushiny GUI. \label{fig:bboushiny}](figures/bboushiny-survival.png)
+
+# Statement of need
+
+Boreal caribou are found in most provinces and territories in Canada and have been listed as threatened since 2003 [@ECCC_2023].
+Each jurisdiction has their own monitoring program responsible for boreal caribou.
+Numerous methods have been used to estimate population growth, which can make cross-jurisdictional conversations on the health of the species complex. 
+
+The suite of R packages in `bbousuite` aim to address this problem.
+`bboutools` provides a standardized methodology with simple, general models and sensible defaults for estimating survival, recruitment and population growth that can be compared across jurisdictions.
+In addition, `bboushiny` provides access to `bboutools` functionality in a GUI; `bbouretro` facilitates comparison with traditional methods; and `bbousims` facilitates comparison of various methods' ability to recover known parameter values from simulated data. 
+
+There is an existing web-based application for estimating boreal caribou population growth rate [@eacker_webbased_2019]. 
+However, users reported that it was difficult to use, especially if not familiar with Bayesian statistics, and had too many options to be considered a standarized method.  
+
+The `caribouMetrics` R package, and associated shiny app `BayesianCaribouDemographicProjection`, also have functionality to estimate boreal caribou demographic parameters.  
+However, the primary goal of these tools is to forecast boreal caribou population under different monitoring and disturbance scenarios [@dyson_2022].
+
+# Overview of methods
+
+Despite key methodological differences, which are explained in more detail below, `bboutools` and `bbouretro` use the same underlying equations to estimate survival, recruitment and population growth. 
+Survival is estimated from the monthly fate of collared adult females using the staggered entry Kaplan-Meier method [@pollock_survival_1989], with an option to include uncertain mortaliites in the total monthly mortalities summed prior to model fitting. 
+Recruitment is estimated from annual composition surveys following methods in DeCesare et al. [-@decesare_estimating_2012], where the adjusted recruitment accounts for recruitment of calves into the yearling/adult age class at the end of the caribou year.
+Population growth ($\lambda$) is estimated using the Hatter-Bergerud method [@hatter_moose_1991].
+Both `bboutools` and `bbouretro` allow the user to adjust the start month of the biological year and data are aggregated by biological year prior to model fitting.
```

Comment:
I think the phrase aggregate is slightly misleading as the survival is still performed on monthly collar numbers.

Reply:
good point - maybe something like:
"adjust the start month of the biological year and year for each observation is reassigned prior to model fitting."